auth/v1.js

```diff
@@ -762,6 +763,57 @@ api.declare({
 });
 
 
+api.declare({
+  method:     'post',
+  route:      '/test-authenticate',
+  name:       'testAuthenticate',
+  input:      'test-authenticate-request.json#',
+  output:     'test-authenticate-response.json#',
+  stability:  'experimental',
+  title:      "Test Authentication",
+  description: [
+    "Utility method to test your implementation of the authentication scheme.",
+    "The request payload carries a set of scopes that the client has",
+    "and a set of scopes that the request is required to have.",
+    "",
+    "You can authenticate against this end-point with `tester` as `clientId`",
+    "and `no-secret` as `accessToken`.",
```

Comment:
Rephrase this a little, since I didn't understand until I read the code:
---
Utility method to test client implementations of TaskCluster authentication.

Rather than using real credentials, this endpoint accepts requests with clientId `tester` and accessToken `no-secret`.  That client's scopes are based on `clientScopes` in the request body.